nixos/modules/services/network-filesystems/ipfs.nix

```diff
@@ -257,7 +257,7 @@ in
       '' + optionalString cfg.autoMigrate ''
         ${pkgs.ipfs-migrator}/bin/fs-repo-migrations -to '${cfg.package.repoVersion}' -y
```

Comment:
Can you put the nice description you gave this PR into the actual commit message so that it isn't lost?

Reply:
Reworded the commit message.